### DISCUSSION TITLE
Add zigbee2mqtt plug-in on bare homeassistant service

### DISCUSSION THREAD

**Discussion by bromnl** [Koenkk/zigbee2mqtt#6974] [General]
I saw a very nice video about zigbee2mqtt on a hassio. They could manage there zigbee2mqtt service from the homeassistant ui. 
Is it possible to install this addon on a bare homeassistant installation?

**Comment by Koenkk** [comment]
AFAIK you need  a supervised Home Assistant installation to run HA addons. However you probably saw the z2m frontend: https://www.zigbee2mqtt.io/information/frontend.html

**Reply by bromnl** [reply]
@Koenkk that was wat I saw an what I need. It would be a nice to have. If it could be integrated in the ha interface, but this separate web app is for me a very nice feature of z2m.

**Reply by DubhAd** [reply]
You can use things like the [website card](https://www.home-assistant.io/lovelace/iframe/) to embed the UI, or the [iframe panel](https://www.home-assistant.io/integrations/panel_iframe/) to put an entry on the menu on the left.
